MangaManager/src/Common/loadedcomicinfo.py

```diff
@@ -313,14 +312,14 @@ def _load_metadata(self):
 
         if xml_string:
             try:
-                self.cinfo_object = parseString(xml_string, silence=True)
+                self.cinfo_object = ComicInfo.from_xml(xml_string)
             except XMLSyntaxError as e:
                 logger.warning(LOG_TAG + f"Failed to parse XML:\n{e}\nAttempting recovery...")
-                try:
-                    self.cinfo_object = parseString(xml_string, doRecover=True, silence=True)
-                except XMLSyntaxError:
-                    logger.error(f"[{'Reading Meta':13s}] Failed to parse XML: {e} - Recovery attempt failed")
-                    raise CorruptedComicInfo(self.file_path)
+                # try:
+                #     self.cinfo_object = ComicInfo.from_xml(xml_string)
+                # except XMLSyntaxError:
+                #     logger.error(f"[{'Reading Meta':13s}] Failed to parse XML: {e} - Recovery attempt failed")
+                #     raise CorruptedComicInfo(self.file_path)
```

Comment:
Might be interesting to have something to "fix" xml data.
Else just clean this up with a "todo"